xplat/web/build.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 set -e
+
+if [ -z $(type -t $function) ]; then
```

Comment:
This gives me `bad option: -t` on the z shell, which is the default now in Catalina... yet another time Apple broke backwards compatibility

Reply:
Oh wait, I just realized that you the shebang at the top will make this run on zsh too. I'll merge it now.